### PR TITLE
Sticky nav UX tests

### PR DESCRIPTION
**Pull request by uplne**
Because sticky nav tests are running only on the US edition and for UX testing we need them to run on UK edition, I've moved tests to its own file and added two extra 0 audition tests just for UX (internal) testing purposes.

This will be reverted after UX test will happen early next week.

CC @kelvin-chappell

### CODE REVIEW

File: common/app/conf/switches.scala [hunk 1]
```diff
@@ -604,6 +604,20 @@ object Switches {
     safeState = Off, sellByDate = new LocalDate(2015, 5, 21)
   )
 
+  val ABMtStAllUxSwitch = Switch(
+    "A/B Tests",
+    "ab-mt-st-all-ux",
+    "Top navigation and top ad slot are sticky with navigation going to slim mode for UX testing",
+    safeState = Off, sellByDate = never
```
Comment (kelvin-chappell) [inline common/app/conf/switches.scala]: Should give these expiry dates so we remember to change the code.

File: common/app/conf/switches.scala [hunk 2]
```diff
@@ -604,6 +604,20 @@ object Switches {
...
+  )
+
+  val ABMtStBurgerUxSwitch = Switch(
+    "A/B Tests",
+    "ab-mt-st-burger-ux",
+    "Top navigation and top ad slot are sticky with navigation going to slim mode for UX testing",
+    safeState = Off, sellByDate = never
```
Comment (kelvin-chappell) [inline common/app/conf/switches.scala]: Should give these expiry dates so we remember to change the code.

**Comment by kelvin-chappell** [comment]
Good idea to split code out and use 0% a/b test.  :+1:

File: common/app/conf/switches.scala (hunk 1 above)
Comment (uplne) [inline common/app/conf/switches.scala]: Damn copy/paste :smile: I gave it 10 days.

**Comment by paperboyo** [comment]
Hi,

Would it be easy/possible to exclude sticky nav from pages that use sticky chapter widget https://github.com/guardian/interactive-page-enhancers?

It clashes with sticky nav, like so: [INTERNAL RESOURCE] https://theguardian.slack.com/archives/guardian-visuals/p1430439281000169

Maybe it’s not worth it during tests (not going to skew results massively), but have to be definitely sorted out before any sticky nav goes to PROD.

Regards
Mateusz

**Comment by Calanthe** [comment]
Hi Mateusz,

We are aware of those issues and we will definitely fix them when there will be a decision to move sticky nav out of the AB tests.

**Comment by paperboyo** [comment]
tumblr_mg2ijwayyl1qhmrpto1_500

**Pull request merged**